src/lib/assetBridger/ethBridger.ts

```diff
@@ -194,6 +201,37 @@ export class EthBridger extends AssetBridger<
     }
   }
 
+  /**
+   * Deposit ETH from L1 onto a different L2 address
+   * @param params
+   * @returns
+   */
+  public async depositTo(
+    params: EthDepositToParams,
+    l2Provider: Provider,
+    gasOverrides?: GasOverrides
+  ): Promise<L1ContractTransaction> {
```

Comment:
I think we want to return `L1ContractCallTransaction` here so we can do `.waitForL2` like in [Erc20Bridger.deposit](https://github.com/OffchainLabs/arbitrum-sdk/blob/main/src/lib/assetBridger/erc20Bridger.ts#L624)

Reply:
Oh, that's right! Yes, definitely. But I'm actually thinking we could change the return type for the `L1ToL2MessageCreator.createRetryableTicket()` . The function hasn't been used until now, so we won't be breaking anything, and it also makes sense to have `.waitForL2` there (plus, I won't need to convert the types in `depositTo()`)
What do you think?

Reply:
It might not be used internally, but it's possible that it's used externally by someone who installed the package, so changing the return type would be a breaking change. :slightly_smiling_face:

Reply:
Oh, I meant that `L1ToL2MessageCreator.createRetryableTicket()` was broken until yesterday ([fix merged here](https://github.com/OffchainLabs/arbitrum-sdk/pull/237)), so it couldn't be used in any way ( unless I'm missing something! ).